articles/azure-app-configuration/quickstart-dotnet-core-app.md

```diff
@@ -62,7 +62,21 @@ You use the [.NET Core command-line interface (CLI)](/dotnet/core/tools/) to cre
     using Microsoft.Extensions.Configuration.AzureAppConfiguration;
     ```
 
-4. Update the `Main` method to use App Configuration by calling the `builder.AddAzureAppConfiguration()` method.
+4. Use App Configuration by calling the `builder.AddAzureAppConfiguration()` method.
+
+    #### [.NET 6.x](#tab/core6x)
+
+    ```csharp
+    var builder = new ConfigurationBuilder();
+    builder.AddAzureAppConfiguration(Environment.GetEnvironmentVariable("ConnectionString"));
+
+    var config = builder.Build();
+    Console.WriteLine(config["TestApp:Settings:Message"] ?? "Hello world!");
+    ```
+
+    #### [.NET Core 3.x](#tab/core3x)
+
+    Update the `Main` method.
```

Comment:
Instead of this extra line in .NET Core 3.x tab, we could just say "Update the code to use App Configuration by calling the `builder.AddAzureAppConfiguration()` method." on line 65

Reply:
look good, let me update it.